README.Rmd

```diff
@@ -66,8 +92,71 @@ compound("path/to/Alfentanil-Model.json")
 
 ### Explore Compound
 
-```{r}
-rifampicin$Formulations
+Once you have imported a compound, you can explore its properties, such as formulations, protocols, and parameters:
```

Comment:
```suggestion
Once you have imported a compound model, you can explore its properties, such as formulations, protocols, and parameters:
```

Reply:
I would replace "model" by "snapshot" in the sentence.